README.md

```diff
@@ -1,10 +1,5 @@
-Coderwall SVG [![Donate on Gittip](http://badgr.co/gittip/twolfson.png)](https://www.gittip.com/twolfson/)
+Coderwall SVG [![Donate on Gittip](https://img.shields.io/gratipay/twolfson.svg)](https://gratipay.com/~twolfson/)
 =============
-In this repo is a quick (and sloppy) conversion of the [Coderwall](http://coderwall.com/) logo to SVG
+In this repo is a quick conversion of the [Coderwall](http://coderwall.com/) logo to SVG
 
-![Coderwall logo](https://raw.github.com/twolfson/coderwall-svg/master/coderwall.png)
-
-Download Links
---------------
-
-[svg](https://raw.github.com/twolfson/coderwall-svg/master/coderwall.svg) [png](https://raw.github.com/twolfson/coderwall-svg/master/coderwall.png)
\ No newline at end of file
+[![Coderwall logo](https://cdn.rawgit.com/denilsonsa/coderwall-svg/88921c6211e60c960cd8c154068e1688affaa97b/coderwall.svg)](coderwall.svg)
```

Comment:
Please move back to `raw.github.com`, this was intentional as:

- There is no caching when viewing the resource meaning `master` is always up to date
- We don't need to worry about bumping the SHA on every edit to the SVG